sdk/src/main/java/com/ciscowebex/androidsdk/phone/CallMembership.java

```diff
@@ -102,9 +102,16 @@
     /**
      * @return The email address of the person in this CallMembership.
      * @since 0.1
+     * @deprecated The email address can no longer get, since privacy protection reason, please use {@link CallMembership#getDisplayName()}
      */
     String getEmail();
 
+    /**
+     * @return The display name of the person in this CallMembership.
+     * @since 2.8.0
+     */
+    String getDisplayName();
```

Comment:
In V3 Ankit has already introduced getName() function. Not sure which one to change?